packages/@dcl/inspector/jest.config.js

```diff
@@ -1,11 +1,22 @@
 module.exports = {
   moduleFileExtensions: ["ts", "js", "jsx", "tsx"],
   transform: {
-    "^.+\\.[tj]sx?$": require.resolve('./jest-transformer')
+    "^.+\\.(js|jsx|ts|tsx)$": ["ts-jest", {
+      tsconfig: "test/tsconfig.json", 
```

Comment:
this makes tests 100x slower. the jest-transformers exists because of that, importing and parsing babylon consumes several minutes before the tests actually start